File: JAVA/DGISoftModel/src/pe/edu/pucp/dgisoft/model/MiembroPUCP.java
package pe.edu.pucp.dgisoft.model;

import java.util.Date;

public class MiembroPUCP {
    private int idMiembroPUCP;
    private char tipoMiembro;
    private String codigoPUCP;
    private String DNI;
    private String nombres;
    private String apellido_paterno;
    private String apellido_materno;
    private char genero;
    private Date fechaNacimiento;

    public int getIdMiembroPUCP() {
        return idMiembroPUCP;
    }

    public void setIdMiembroPUCP(int idMiembroPUCP) {
        this.idMiembroPUCP = idMiembroPUCP;
    }

    public char getTipoMiembro() {
        return tipoMiembro;
    }

    public void setTipoMiembro(char tipoMiembro) {
        this.tipoMiembro = tipoMiembro;
    }

    public String getCodigoPUCP() {
        return codigoPUCP;
    }

    public void setCodigoPUCP(String codigoPUCP) {
        this.codigoPUCP = codigoPUCP;
    }

    public String getDNI() {
        return DNI;
    }

    public void setDNI(String DNI) {
        this.DNI = DNI;
    }

    public String getNombres() {
        return nombres;
    }

    public void setNombres(String nombres) {
        this.nombres = nombres;
    }

    public String getApellido_paterno() {
        return apellido_paterno;
    }

    public void setApellido_paterno(String apellido_paterno) {
        this.apellido_paterno = apellido_paterno;
    }

    public String getApellido_materno() {
        return apellido_materno;
    }

    public void setApellido_materno(String apellido_materno) {
        this.apellido_materno = apellido_materno;
    }

    public char getGenero() {
        return genero;
    }

    public void setGenero(char genero) {
        this.genero = genero;
    }

    public Date getFechaNacimiento() {
        return fechaNacimiento;
    }

    public void setFechaNacimiento(Date fechaNacimiento) {
        this.fechaNacimiento = fechaNacimiento;
    }
    
    
}

File: CSharp/DGISoft/DGISoftModel/MiembroPUCP.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DGISoftModel
{
    public class MiembroPUCP
    {
        private int _idMiembroPUCP;
        private char _tipoMiembro;
        private String _codigoPUCP;
        private String _DNI;
        private String _nombres;
        private String _apellido_paterno;
        private String _apellido_materno;
        private char _genero;
        private DateTime _fechaNacimiento;
        public int IdMiembroPUCP { get => _idMiembroPUCP; set => _idMiembroPUCP = value; }
        public char TipoMiembro { get => _tipoMiembro; set => _tipoMiembro = value; }
        public string CodigoPUCP { get => _codigoPUCP; set => _codigoPUCP = value; }
        public string DNI { get => _DNI; set => _DNI = value; }
        public string Nombres { get => _nombres; set => _nombres = value; }
        public string Apellido_paterno { get => _apellido_paterno; set => _apellido_paterno = value; }
        public string Apellido_materno { get => _apellido_materno; set => _apellido_materno = value; }
        public char Genero { get => _genero; set => _genero = value; }
        public DateTime FechaNacimiento { get => _fechaNacimiento; set => _fechaNacimiento = value; }
    }
}